develop/toolkit/interoperability/paraspell-xcm-sdk/transfer-tokens-between-parachains.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Transfer Tokens Between Parachains
+description: A step-by-step guide to using the ParaSpell XCM SDK to build, verify, and execute a transfer from one Parachain to another.
+---
+
+# Transfer Tokens Between Parachains
+
+## Introduction
+
+This guide walks you through transferring tokens between two parachains using the [ParaSpell XCM SDK](https://paraspell.github.io/docs/){target=\_blank}. This example utilizes [Asset Hub](/polkadot-protocol/architecture/system-chains/asset-hub/){target=\_blank} and the [People Chain](/polkadot-protocol/architecture/system-chains/people/){target=\_blank}. However, the same approach can be applied to transfers between other parachains.
+
+For development purposes, this guide will use the [Polkadot TestNet](/develop/networks/#paseo){target=\_blank}, so the transferred token will be PAS.
+
+In this guide, you’ll learn how to:
+
+- Build an XCM transfer transaction using ParaSpell XCM SDK.
+- Perform a dry run to validate the transfer.
+- Verify the [Existential Deposit (ED)](/polkadot-protocol/glossary/#existential-deposit){target=\_blank} requirement on the destination chain.
+- Retrieve information regarding the transfer, along with fee estimates.
+- Submit the transaction.
+
+### Prerequisites
+
+- Knowledge of the [fundamentals of Polkadot](/polkadot-protocol/parachain-basics/){target=\_blank}
+- Basic understanding of [XCM](/develop/interoperability/intro-to-xcm/){target=\_blank}
+- Basic familiarity with JavaScript/TypeScript
+- Install [bun](https://bun.com/docs/installation){target=\_blank}, a JavaScript/TypeScript package manager
+
+## Initialize Your Project
+
+Create the project folder:
+
+```bash
+mkdir paraspell-transfer
+cd paraspell-transfer
+```
+
+Initialize the JavaScript project:
+
+```bash
+bun init -y
+```
+
+Install the required dependencies:
+
+```bash
+bun add @paraspell/sdk@11.3.2 polkadot-api@1.17.1 @polkadot-labs/hdkd-helpers@0.0.25 @polkadot-labs/hdkd@0.0.24
```

Comment:
```suggestion
bun add @paraspell/sdk@11.3.2 polkadot-api@1.17.1 @polkadot-labs/hdkd-helpers@0.0.25 @polkadot-labs/hdkd@0.0.24
```
@brunopgalvao please open a ticket to see about adding these to variables.yml and using placeholders so these versions are easier to keep up to date. Not a blocker to ship this now though.